### PR TITLE
feat: scaffolder job progress logs in full screen.

### LINKED ISSUE #3169: feature(scaffolder-ui): Full screen logs.
Ability to view these logs in full screen.

Screenshot from 2020-10-30 10-14-48

## Possible Implementation

Adding additional component [here](https://github.com/spotify/backstage/tree/master/plugins/scaffolder/src/components/JobStage) which may be another ```Dialogue``` but full screen.

## Context

Something like this one?

Screenshot from 2020-10-30 10-31-20

### PR DESCRIPTION
**Pull request by Marvin9**
## Hey, I just made a Pull Request!

Resolves #3169 .

Screenshot from 2020-11-02 21-53-28

Screenshot from 2020-11-02 22-28-36

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/spotify/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

File: plugins/scaffolder/src/components/JobStage/JobStage.tsx [hunk 1]
```diff
@@ -130,8 +136,20 @@ export const JobStage = ({ endedAt, startedAt, name, log, status }: Props) => {
...
+              open={logsFullScreen}
+              onClose={toggleLogsFullScreen}
+              log={log}
+            />
+            <div
+              style={{ height: '20vh', width: '100%' }}
+              onClick={toggleLogsFullScreen}
+            >
+              <LazyLog
+                text={`${log.join('\n')}\nclick to view logs in full screen.`}
```
Comment (freben) [inline plugins/scaffolder/src/components/JobStage/JobStage.tsx:149]: Could you instead make something along the lines of a `position: absolute` div in the upper right corner of the log text that holds a magnifying glass icon button? I would find that a lot clearer

```
,--------------------------------,
| Log Text                ,----, |
| Log Text                | O  | |
|                         |  \ | |
|                         `----` |
|                                |
|                                |
```

**Comment by freben** [comment]
@Marvin9 Hey! Just checking in to see how things are and whether the feedback made sense

**Comment by Marvin9** [comment]
@freben Sorry I am on tight academic schedule. I will try to complete it ASAP. All good from your side.

**Comment by benjdlambert** [comment]
Hey @Marvin9! Happy New Year :tada: Is there anything we can do to help here?

**Comment by Marvin9** [comment]
Happy new year! Sorry for inconvenience. I appreciate If you can take this up. I may not spare time to complete in near future.

File: plugins/scaffolder/src/components/JobStage/JobStage.tsx (hunk 1 above)
Comment (benjdlambert) [reply plugins/scaffolder/src/components/JobStage/JobStage.tsx]: I opted for more of a MUI style using the `AccordionDetails` with a button to maximize the logs for now.

We can revisit the frontend outside of this PR. :)

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**